### ISSUE TITLE
publish-to-pypi workflow fails with PYPI_API_TOKEN auth issue

### ISSUE THREAD

**Issue by rnikutta** [lincc-frameworks/python-project-template#199]
Hi LINCC team,

I'm trying out the-python-project template for a new project, and got pretty far. But the publish-to-pypi.yml workflow fails, despite having created PYPI_API_TOKEN as described in https://lincc-ppt.readthedocs.io/en/latest/practices/pypi.html

Here's the failed step:

https://github.com/rnikutta/astrograppa/actions/runs/5045542456/jobs/9049948938

I confirm that:
- I have a pypi account
- I created an (org-wide) API token on pypi
- I've created a new "codespaces" token on Github, and named it PYPI_API_TOKEN
- Its value is the pypi-generated token, including the 'pypi-' prefix.
- The project has zero own additions for now, only the hydrated template.

When I trigger a new release on Github, it builds the package etc., but then fails with the message about auth issues.

What could be the issue? I don't see anything about it in the open or closed Issues here on Github.

Thanks!
Robert

**Comment by rnikutta** [comment]
Update: your docs (https://lincc-ppt.readthedocs.io/en/latest/practices/pypi.html) said clearly "Save the API token in your repository as an “Action Secret” following these instructions." The link to GH under "these instructions" (https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/managing-encrypted-secrets-for-your-repository-and-organization-for-github-codespaces#adding-secrets-for-a-repository) says to select "Codespaces", and that's what I did initially. Sorry!

tl;dr: it works now, thanks! Closing this ticket.
